pkg/utils/meta.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func SetMeta(apiClient *client.APIClient, r *schema.ResourceData) {
-	apiClient.SetMeta(nil, func(ctx *context.Context, meta interface{}) {
+	meta_err := apiClient.SetMeta(nil, func(ctx *context.Context, meta interface{}) {
```

Comment:
do not use multiple error variable names, please common convention of either err or meta_err

Reply:
Done